src/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -118,7 +132,44 @@ public void onAttach(Activity activity) {
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        Log_OC.i(TAG, "onCreateView() start");
+        View v = super.onCreateView(inflater, container, savedInstanceState);
+
+        // Setup FAB listeners
```

Comment:
DONE - onActivityCreated is the right one, see https://github.com/owncloud/android/commit/e4d936e83754b823c90160889ce4704dfc3bbe61